nifi/Dockerfile

```diff
@@ -63,6 +63,47 @@ rm -rf /stackable/nifi-${PRODUCT}/docs
 chmod -R g=u /stackable
 EOF
 
+FROM stackable/image/java-devel AS nifi-iceberg-bundle-builder
+
+ARG NIFI_ICEBERG_BUNDLE
+ARG PRODUCT
+ARG STACKABLE_USER_UID
+
+USER ${STACKABLE_USER_UID}
+WORKDIR /build
+
+RUN <<EOF
+mkdir -p /stackable
+
+# NiFI 1.x natively supports Iceberg, no need to build an iceberg-bundle for it
+if [[ "${PRODUCT}" != 1.* ]] ; then
+    curl "https://github.com/stackabletech/nifi-iceberg-bundle/archive/refs/tags/${NIFI_ICEBERG_BUNDLE}.tar.gz" | tar -xzC .
```

Comment:
I _think_ we probably want a script to upload this to Nexus, and pull from there, no?

Reply:
I copied the mechanism from Hadoop:
* https://github.com/stackabletech/docker-images/blob/5bf51497b7c4d903a94f24f8e144042f02ff487c/hadoop/Dockerfile#L133

A quick search also brought up:

* https://github.com/stackabletech/docker-images/blob/5bf51497b7c4d903a94f24f8e144042f02ff487c/hbase/Dockerfile#L107
* https://github.com/stackabletech/docker-images/blob/14a6c24e3be0f3c894959fd9d18501455298a947/opa/Dockerfile#L28

I'm personally happy with that, and would prefer to prevent the indirection. However, this is not a strong opinion, please feel to object :)

Reply:
Oh sorry, I missed that the repository was under `stackabletech`.

As you were...